corehq/sql_db/connections.py

```diff
@@ -79,32 +80,36 @@ def is_citus_db(self):
 class ConnectionManager(object):
     """
     Object for dealing with sqlalchemy engines and sessions.
+
+    Maintains a mapping of `engine_id` to Django db alias as well as load balancing configuration
+    from the `REPORTING_DATABASES` setting.
     """
 
     def __init__(self):
         self._session_helpers = {}
-        self.db_connection_map = {}
         self.read_database_mapping = {}
         self.engine_id_django_db_map = {}
         self._populate_connection_map()
 
-    def _get_or_create_helper(self, engine_id):
-        if engine_id not in self._session_helpers:
-            self._session_helpers[engine_id] = SessionHelper(self.get_connection_string(engine_id))
-        return self._session_helpers[engine_id]
+    def _get_or_create_helper(self, connection_string):
+        if connection_string not in self._session_helpers:
```

Comment:
Assuming this is just the name /key, if thats correct, is `connection_name` or `connection_`alias` a better name here?

Reply:
It's actually the real connection string that gets passed to the driver.